### PR TITLE
Adding outputs for WaaS

### PR DESCRIPTION
**Pull request by shishir-msft**

### CODE REVIEW

**Comment by shishir-msft** [comment]
Test comment.

**Comment by UmakanthOS** [comment]
Flagging with @naioja. Changes look OK to me.

**Review by venu-sivanadham — CHANGES_REQUESTED** [review]

File: azuredeploy.json [hunk 1]
```diff
@@ -903,29 +899,192 @@
...
+                "wpVersion": "[variables('lampCommon').wpVersion]"
+            }
         },
-        "loadBalancerSku": {
-            "type": "string",
-            "value": "[variables('lampCommon').lbSku]"
+        "webNodesProfile": {
+            "type": "object",
+            "value": {
+                "nodeResourceIds": "[resourceId('Microsoft.Compute/virtualMachineScaleSets', variables('lampCommon').vmssName)]"
```
[the remark this answers is not in the document]
Comment (shishir-msft) [reply azuredeploy.json:1035]: Yes I did. SKUs are tested along with these changes included.

File: nested/nfs-ha.json [hunk 2]
```diff
@@ -579,5 +579,11 @@
         "nfsHaLbRuleName": "nfs-ha-lb-rule",
         "nfsHaLbProbeName": "nfs-ha-lb-probe",
         "nfsHaLbProbeId": "[resourceId('Microsoft.Network/loadBalancers/probes', variables('nfsHaLbName'), variables('nfsHaLbProbeName'))]"
+    },
+    "outputs": {
+        "nfsHaLbName": {
+            "type": "string",
+            "value": "[variables('nfsHaLbName')]"
+        }
```
[the remark this answers is not in the document]
Comment (shishir-msft) [reply nested/nfs-ha.json]: Thanks for pointing out. Reverted.

File: azuredeploy.json [hunk 3]
```diff
@@ -903,29 +899,192 @@
...
-            "value": "[variables('lampCommon').dbLoginPassword]"
-        },
         "firstFrontendVmIP": {
             "type": "string",
             "value": "[reference('scaleSetTemplate').outputs.webvm1IP.value]"
         },
-        "loadBalancerDNS": {
-            "type": "string",
-            "value": "[variables('lampCommon').lbDns]"
+        "lampCommon": {
```
[the remark this answers is not in the document]
Comment (shishir-msft) [reply azuredeploy.json:906]: I'll take it in a separate PR since it is in scope for demo today.

**Review by venu-sivanadham — APPROVED** [review]

**Pull request merged**